transmogrifier/sources/xml/marc.py

```diff
@@ -972,6 +954,47 @@ def get_publication_frequency(cls, source_record: Tag) -> list[str] | None:
             )
         ] or None
 
+    @classmethod
+    def get_publishers(cls, source_record: Tag) -> list[timdex.Publisher] | None:
+        publishers = []
+        for publisher_marc_field in ["260", "264"]:
+            publishers.extend(
+                [
+                    timdex.Publisher(
+                        name=publisher_name.rstrip(".,") if publisher_name else None,
+                        date=publisher_date.rstrip(".,") if publisher_date else None,
```

Comment:
Good update to `.rstrip`, should have caught that given the `℗2022,` in the all fields test

Reply:
Writing and running unit tests per field method really allows us to pick up on these small things that are easily missed when scrolling through the lengthy assertions of the older tests. 🤓